selene/schema/overrides/mutations.py

```diff
@@ -156,12 +168,15 @@ class ModifyOverrideInput(graphene.InputObjectType):
     hosts = graphene.List(
         graphene.String, description="A list of hosts addresses"
     )
+    new_severity = graphene.Float(
+        required=True, description="Severity to which should be overridden"
+    )
```

Comment:
```suggestion
    new_severity = graphene.Field(
        SeverityType,
        required=True, description="Severity to which should be overridden"
    )
```

Reply:
Implemented with additions in https://github.com/greenbone/hyperion/pull/85/commits/e6f48dc60115afd3a13598aead75bab52a978516